src/common/headlessService/utils.ts

```diff
@@ -106,14 +109,25 @@ export function isEventSelectedCarousel(
   );
 }
 
+export function isLocationsSelected(
+  module: PageModule | PageSidebarModule,
+): module is LocationsSelected {
+  return (
+    (<LocationsSelected>module).locations !== undefined &&
+    // eslint-disable-next-line no-underscore-dangle
+    module.__typename === 'LocationsSelected'
+  );
+}
+
 export function isEventModule(
   module: PageModule | PageSidebarModule,
 ): module is EventModule {
   return (
     isEventSearch(module) ||
     isEventSearchCarousel(module) ||
     isEventSelected(module) ||
-    isEventSelectedCarousel(module)
+    isEventSelectedCarousel(module) ||
+    isLocationsSelected(module)
```

Comment:
Shouldn't this be in function `isVenueModule`? I think it should go so that when a module is EventModule, it needs the LinkedEvents Apollo-client and connection. When it is a VenueModule, it needs the Venue (Tprek) Apollo-client and connection.